docs/content/stack/configuration-variables.md

```diff
@@ -128,6 +128,12 @@ This is the domain to use which is tacked on to the end of the projects url.
 
 Allow disabling the DNS resolver configuration (in case there are issues with it). Set to `true` to activate.
 
+### DOCKSAL_HEALTHCHECK_TIMEOUT
+
+`Default: 60`
+
+How many seconds to give project containers before `fin project start` stops waiting for them to reach healthy state. (Should divide by 5)
```

Comment:
```suggestion
How many seconds to give project containers to reach `healthy` state before `fin` considers the stack startup as failed. (Should be a multiple of 5)
```

I replaced `fin project start` with `fin`, because we should (and will) have the healthcheck done during restart/reset as well, not just `start`.